test/tick-processor/test-tick-processor-polyfill-brokenfile.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (!common.enoughTestCpu)
+  common.skip('test is CPU-intensive');
+
+if (common.isCPPSymbolsNotMapped) {
+  common.skip('C++ symbols are not mapped for this os.');
```

Comment:
Nit: Operating system should be upper cased as in `OS`.

Reply:
Thank you for the advice.It should be upper cased.